app/javascript/packages/document-capture/components/acuant-capture-canvas.jsx

```diff
@@ -188,6 +188,8 @@ function AcuantCaptureCanvas({
       <div id="acuant-sdk-capture-view">
         <canvas
           id="acuant-video-canvas"
+          tabIndex={0}
+          aria-label="Live video from device camera"
```

Comment:
(assuming this will be i18n'd eventually but it's still in draft mode?)

Reply:
> (assuming this will be i18n'd eventually but it's still in draft mode?)

Yep 👍 

I should stop being lazy about this, since at some point it'll bite me 😅 

Though on the other hand, it'd be nice if we could get some automated checks for translated values. Might be tricky to differentiate which attributes ought to be translated though.

Reply:
How easy is it to lint on JSX expressions? We could check the inner HTML values to make sure they're translated, and then we probably list a few common attributes like `aria-label` and `title` and check those and make sure they're not hardcoded?

Reply:
> How easy is it to lint on JSX expressions? We could check the inner HTML values to make sure they're translated, and then we probably list a few common attributes like `aria-label` and `title` and check those and make sure they're not hardcoded?

Fairly easy, actually.

Example: https://astexplorer.net/#/gist/b0c895ee1b4349d2cdd4bacdca494404/f39686b0be6f910da46226e7920441213771e5ff

I guess I was more worried about what exactly we want to be localized, but like you mention with a few of the more common attributes, maybe it's fine to be imperfect.